content/blog/face_detection_with_actix_web.md

```diff
@@ -789,6 +789,10 @@ fn return_overlay(
                 .headers_mut()
                 .insert(CONTENT_TYPE, HeaderValue::from_static("image/jpeg"));
             response
+            // alternatively, use the builder pattern for response
```

Comment:
I think just replace this entirely with what you've written if this works OK.  Much more succinct!

Reply:
I think it would be fine to show both.